src/force.rs

```diff
@@ -0,0 +1,76 @@
+use maths_rs::{length, Vec2d};
+
+use crate::{constants::G, types::Body};
+
+/// Calculates the gravitational force between two masses.
+///
+/// usage
+/// ```rust
+/// use kepler_core::force::gravity_force;
+///
+/// let mass_earth = 5.972e24;
```

Comment:
We may want to use our constants for these values. I tested it locally with `MASS_EARTH` and it seems to work. We can do it after merging in the PR though. 👍🏻